### PR TITLE
Creating a separate directory for storing Ballerina Integrator templates

### PR DESCRIPTION
**Pull request by IrushiL**
> Creating a basic Hello World Service template.

### CODE REVIEW

File: templates/hello_world_service/Ballerina.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[project]
+org-name= "wso2"
+version= "1.0.0"
+license= ["Apache-2.0"]
+authors= ["WSO2"]
+keywords= ["ballerina", "integrator"]
```
Comment (Asitha) [inline templates/hello_world_service/Ballerina.toml]: Shall we have a keyword like `ballerina integrator template`?

File: templates/hello_world_service/src/hello_world_service/tests/main_test.bal [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import ballerina/test;
```
Comment (Asitha) [inline templates/hello_world_service/src/hello_world_service/tests/main_test.bal]: License header?

Comment (IrushiL) [reply templates/hello_world_service/src/hello_world_service/tests/main_test.bal]: Done

**Comment by ThishaniLucas** [comment]
Why should we commit the .keep files? Is there any specific reason for having those folders?

**Comment by IrushiL** [comment]
@ThishaniLucas Since this will be used to create a module from a template, we should keep the folder structure similar to what we would get if we created a default module in Ballerina. The .keep files are committed to ensure the folders are preserved as it is.

**Pull request merged**